src/main/java/gregification/proxy/ExNihiloCommonProxy.java

```diff
@@ -31,8 +25,8 @@ public void preInit() {
     }
 
     @Optional.Method(modid = GFValues.MODID_EXNI)
-    @SubscribeEvent
-    public static void registerRecipes(RegistryEvent.Register<IRecipe> event) {
+    @EventHandler
+    public void init() {
```

Comment:
Doesn't this need to stay as the IRecipe Register event? In `init` CT will not be able to touch them, and I don't believe the material handlers will be able to run either

Reply:
Little typo there, doesn't even need an `@EventHandler` annotation. 

But I removed the `RegistryEvent.Register<IRecipe>` event because ExNihilo recipes are generated/processed in init so the `RegistryEvent` listener would be too early. And yes, `ExNihiloRecipes.registerHandlers()` is meant to be in `ExNihiloMaterialHandler`

Reply:
Will follow up with another PR, unless you fixed it in your local.